### DISCUSSION TITLE
Shouldn't ".destroy()" also remove all event listeners?

### DISCUSSION THREAD

**Discussion by minorbug** [simonwep/viselect#120] [Ideas]
I ran into an issue on a VueJS app that had listeners defined using `on()` but used anonymous functions. When the view was swapped out and back into the page (using vue router), the listeners were added a second time.

I saw an example Vue project that used destroy, but didnt mention cleaning up events. I solved it on my side by specifically defining the functions in my component and using `off()` and each one in the component's `beforeDestroy()` call. But if there isn't any good reason/example to keep these events attached after a call to `destroy()`, can we have SelectionJS handle this for us? When I see `destroy()` and its description, I assume that events are also destroyed.

**Comment by simonwep** [comment]
Gosh - you're right. I somewhat never took care of cleaning them up. Fixed in 6b02489f9283de55130418ba9ae07d279cac519f, I'll release a new patch version this weekend :) Thank you for raising this issue!
